packages/runtime/container-runtime/src/summary/summaryFormat.ts

```diff
@@ -150,6 +150,7 @@ export const metadataBlobName = ".metadata";
 export const chunksBlobName = ".chunks";
 export const electedSummarizerBlobName = ".electedSummarizer";
 export const blobsTreeName = ".blobs";
+export const idCompressorBlobName = ".idCompressor";
```

Comment:
You should add this to `nonDataStorePaths` below. I see that some of the other blob types are not added which is probably an oversight.